### PR TITLE
[v7r3] Remove system logging

### PR DESCRIPTION
**Pull request by fstagni**
As discussed in https://indico.cern.ch/event/854185/

BEGINRELEASENOTES

*Framework:
CHANGE: Removed completely the SystemLogging

ENDRELEASENOTES

### CODE REVIEW

**Comment by chaen** [comment]
As of v7 we can completely remove the ConfReference folder I believe. Right @andresailer

**Comment by fstagni** [comment]
I deleted both already

**Comment by andresailer** [comment]
> As of v7 we can completely remove the ConfReference folder I believe. Right @andresailer

I think only once this is finished: https://github.com/DIRACGrid/DIRAC/issues/4087

**Comment by aldbr** [comment]
Oh sorry, I didn't check properly

**Comment by atsareg** [comment]
Do we have in the ES logging a possibility to create reports on Top Error Messages as used to be in the SystemLogging ?

**Comment by fstagni** [comment]
Sure, you can do all the aggregations you want,e.g. see this dashboard https://es-lhcb-dev.cern.ch/kibana/app/kibana#/dashboard/3b53b830-41b1-11e9-97b4-bda41e0e0b3d?_g=()&_a=(description:'',filters:!(),fullScreenMode:!f,options:(darkTheme:!f,hidePanelTitles:!f,useMargins:!t),panels:!((gridData:(h:6,i:'1',w:4,x:3,y:0),id:'44608810-41af-11e9-92db-b12061b7ddd0',panelIndex:'1',type:visualization,version:'6.2.4'),(gridData:(h:3,i:'2',w:5,x:7,y:0),id:cd447a10-41af-11e9-92db-b12061b7ddd0,panelIndex:'2',type:visualization,version:'6.2.4'),(gridData:(h:3,i:'3',w:6,x:0,y:6),id:f4e760b0-41ae-11e9-92db-b12061b7ddd0,panelIndex:'3',type:visualization,version:'6.2.4'),(gridData:(h:3,i:'4',w:5,x:7,y:3),id:b18a4830-41b0-11e9-92db-b12061b7ddd0,panelIndex:'4',type:visualization,version:'6.2.4'),(gridData:(h:3,i:'5',w:6,x:0,y:9),id:'1a1f8180-41b1-11e9-97b4-bda41e0e0b3d',panelIndex:'5',type:visualization,version:'6.2.4'),(embeddableConfig:(vis:(params:(sort:(columnIndex:4,direction:desc)))),gridData:(h:4,i:'6',w:12,x:0,y:12),id:'96851900-41b2-11e9-92db-b12061b7ddd0',panelIndex:'6',type:visualization,version:'6.2.4'),(gridData:(h:3,i:'7',w:6,x:6,y:9),id:'1e7a01d0-466e-11e9-92db-b12061b7ddd0',panelIndex:'7',type:visualization,version:'6.2.4'),(embeddableConfig:(vis:(legendOpen:!t)),gridData:(h:3,i:'8',w:6,x:6,y:6),id:'7d982120-4a28-11e9-92db-b12061b7ddd0',panelIndex:'8',type:visualization,version:'6.2.4'),(gridData:(h:3,i:'9',w:3,x:0,y:0),id:ef8fb8c0-4af4-11e9-92db-b12061b7ddd0,panelIndex:'9',type:visualization,version:'6.2.4'),(gridData:(h:3,i:'10',w:3,x:0,y:3),id:'56fabf60-4af4-11e9-92db-b12061b7ddd0',panelIndex:'10',type:visualization,version:'6.2.4')),query:(language:lucene,query:''),timeRestore:!f,title:'DIRAC%20Logs',viewMode:view)

which is for the DIRAC certification setup.

File: src/DIRAC/Core/scripts/install_full.cfg [hunk 1]
```diff
@@ -141,10 +138,7 @@ LocalInstallation
    Services += Transformation/TransformationManager
 
    # The list of Agents to be installed
-   Agents  = Framework/SystemLoggingDBCleaner
-   #Agents += Framework/CAUpdateAgent
-   Agents += Framework/TopErrorMessagesReporter
-   Agents += Framework/ErrorMessageMonitor
+   Agents  = Framework/CAUpdateAgent
```
[the remark this answers is not in the document]
Comment (fstagni) [reply src/DIRAC/Core/scripts/install_full.cfg:141]: But I think it should be there.

**Review by atsareg — APPROVED** [review]

**Pull request merged**